src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceActions/ResourceDropdown.html

```diff
@@ -1,6 +1,6 @@
 <div class="dropdown">
-    <div class="btn btn-default dropdown-toggle" aria-haspopup="isShowDropdown" aria-expanded="isShowDropdown">
-        <i class="icon-document moving-column-icon" data-ng-click="toggleDropdown()"></i>
+    <div aria-haspopup="isShowDropdown" aria-expanded="isShowDropdown">
+        <i class="icon-three-dots dropdown-icon" data-ng-if="!data.modals.modal" data-ng-click="data.toggleDropdown()"></i>
```

Comment:
the `data-ng-if` and `data-ng-click` conditions actually belong to the next commit. sorry for the confusion.